generators/iac-ci/templates/build/module.codebuild.tf

```diff
@@ -0,0 +1,14 @@
+module "codebuild" {
+  source                 = "jameswoolfenden/codebuild/aws"
```

Comment:
Assuming this points to an external module yeah?

Reply:
Yup, it uses 'source' and 'version' to pick up the module of that particular release from James's github